site/content/integrate/slash-commands/_index.md

```diff
@@ -79,11 +79,13 @@ Slash command responses support more than just the `text` field. Here is a full
 | text | [Markdown-formatted](https://docs.mattermost.com/help/messaging/formatting-text.html) message to display in the post. | If `attachments` is not set, yes |
 | response\_type | Set to blank or `ephemeral` to reply with a message that only the user can see. <br> Set to `in_channel` to create a regular message.<br> Defaults to `ephemeral`. | No |
 | username | Overrides the username the message posts as.<br> Defaults to the username set during webhook creation or the webhook creator's username if the former was not set.<br> Must be enabled [in the configuration](https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames). | No |
+| channel\_id | Overrides the channel the message posts to.<br> Defaults to the channel the command was issued in. | No |
```

Comment:
grammar nit: trailing prepositions make for awkward English.
```suggestion
| channel\_id | Overrides the channel to which the message posts.<br> Defaults to the channel in which the command was issued. | No |
```

Reply:
agreed